README.md

```diff
@@ -18,6 +18,7 @@ const myAppOptions = {
   ...APIOptions,
   clientSecret: 'my-client-secret'
   clientId: 'my-super-secret-app-id',
+  withEmailScope: false, // only supported for non-embedded apps
```

Comment:
My pipe dream is that node-private will be gone and well have a decent auth system before that happens =(